utils/zerovec/src/map2d/mod.rs

```diff
@@ -19,6 +19,23 @@ use crate::map::ZeroMapKV;
 use crate::map::{MutableZeroVecLike, ZeroVecLike};
 pub use borrowed::ZeroMap2dBorrowed;
 
+// ZeroMap2d contains 4 fields:
```

Comment:
nit: the explainer comment should probably be right before `struct` so that it's visible nearby in the source